tests/core/CreateOperation.spec.ts

```diff
@@ -23,41 +27,98 @@ describe('CreateOperation', async () => {
       expect(result.delta).toBeUndefined();
+      expect(result.suffixData).toBeDefined();
     });
 
     it('should throw sidetree error if object contains more or less than 3 properties', () => {
       const twoProperties = { one: 1, two: 2 };
       const fourProperties = { one: 1, two: 2, three: 3, four: 4 };
 
       try {
-        CreateOperation.parseJcsObject(twoProperties, Buffer.from(JSON.stringify(twoProperties)));
+        CreateOperation.parseJcsObject(twoProperties, Buffer.from(JSON.stringify(twoProperties)), false);
         fail('expect to throw sidetree error but did not');
       } catch (e) {
         expect(e).toEqual(new SidetreeError(ErrorCode.CreateOperationMissingOrUnknownProperty));
       }
 
       try {
-        CreateOperation.parseJcsObject(fourProperties, Buffer.from(JSON.stringify(fourProperties)));
+        CreateOperation.parseJcsObject(fourProperties, Buffer.from(JSON.stringify(fourProperties)), false);
         fail('expect to throw sidetree error but did not');
       } catch (e) {
         expect(e).toEqual(new SidetreeError(ErrorCode.CreateOperationMissingOrUnknownProperty));
       }
     });
 
     it('should throw sidetree error if type is not create', () => {
-      const testObject = { type: 'notCreate', two: 2, three: 3 };
+      const testObject = { 
+        type: 'notCreate',
+        suffix_data: {
+          delta_hash: 'something',
+          recovery_commitment: 'something',
+          type: 'type'
+        },
+        delta: 'something'
+      };
+
+      spyOn(CreateOperation as any, 'validateSuffixData').and.callFake(() => {
+        // do nothing
+      });
+
       try {
-        CreateOperation.parseJcsObject(testObject, Buffer.from(JSON.stringify(testObject)));
+        CreateOperation.parseJcsObject(testObject, Buffer.from(JSON.stringify(testObject)), false);
         fail('expect to throw sidetree error but did not');
       } catch (e) {
         expect(e).toEqual(new SidetreeError(ErrorCode.CreateOperationTypeIncorrect));
       }
     });
+
+    it('should throw sidetree error if has more or less than 1 property when in anchor file mode', () => {
+      const testObject = { 
+        type: 'this should not exist',
+        suffix_data: {
+          delta_hash: 'something',
+          recovery_commitment: 'something',
+          type: 'type'
+        }
+      };
+      try {
+        CreateOperation.parseJcsObject(testObject, Buffer.from(JSON.stringify(testObject)), true);
+        fail('expect to throw sidetree error but did not');
+      } catch (e) {
+        expect(e).toEqual(new SidetreeError(ErrorCode.CreateOperationMissingOrUnknownProperty));
+      }
+    })
   });
+
+  describe('computeJcsDidUniqueSuffix', () => {
+    it('should return expected did unique suffix', () => {
+      const actual = Multihash.canonicalizeThenHashThenEncode(suffixData);
+      const expected = fs.readFileSync('tests/fixtures/uniqueSuffix/resultingSuffix.txt', 'utf8');
```

Comment:
Style nit: I can see the argument for external files. Though I'd suggest to encapsulate/hide info when possible, in this case I suggest to have both JSON inline instead of external file 'coz those data are not used anywhere.